exe/node/Main.hs

```diff
@@ -57,8 +57,7 @@ main = do
     nid          <- pure (mkNodeId $ fst keys)
     mem          <- Mempool.newIO
     gen          <- Yaml.decodeFileThrow (genesisPath optPaths) :: IO GenesisBlock
-    genState     <- either (die . fromEvalError) pure $
-                        evalBlock Rad.txEval Rad.pureEnv gen
+    let (genState, _receipts) = evalBlock Rad.txEval Rad.pureEnv gen
```

Comment:
On a second thought: it doesn't seem wise to ignore evaluation errors for the genesis - there is no guarantee that the file at the given path decodes to a valid block.

Reply:
A genesis block can be valid even if it contains transactions that produce evaluation errors. In that respect it is no different than any other block.

I agree however that we also need to validate the genesis block, in particular that the state hash matches the produced state. With #486 this code here will be moved to the ledger which will take care of that validation.

Reply:
:thinking: 
Independent of this PR, since we basically decide what goes in genesis though, we should probably validate that it has no evaluation errors.